R/provider-ollama.R

```diff
@@ -40,7 +40,7 @@ chat_ollama <- function(system_prompt = NULL,
   chat_openai(
     system_prompt = system_prompt,
     turns = turns,
-    base_url = base_url,
+    base_url = file.path(base_url, "v1"),
```

Comment:
Would you mind adding a comment here like "for openAI compatible API"?

Reply:
Sounds good.